src/Engine/Geometry/MathFunctions.cpp

```diff
@@ -1,8 +1,8 @@
-#include "MathFunctions.hpp"
 #include <glm/gtc/quaternion.hpp>
+#include "MathFunctions.hpp"
 
 float Geometry::DotQuat(const glm::quat& q1, const glm::quat& q2) {
-    return q1.x*q2.x + q1.y*q2.y + q1.z*q2.z + q1.w*q2.w;
+    return q1.x * q2.x + q1.y * q2.y + q1.z * q2.z + q1.w * q2.w;
 }
 
 void Geometry::NormalizeQuat(glm::quat& q) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'NormalizeQuat' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)